test/test_smtlib.cpp

```diff
@@ -372,3 +373,147 @@ TEST(TestSMTLib, TestBitVectorArithmeticXor) {
 
     ASSERT_EQ(ss.str(), output);
 }
+
+TEST(TestSMTLib, TestModelInt) {
+    using namespace logicbase;
+
+    std::stringstream ss;
+
+    std::string output = "(\n"
+                         "(define-fun x () Int 8)\n"
+                         "(define-fun y () Int 6)\n"
+                         ")\n";
+
+    smtliblogic::SMTLibLogicModel model(output);
+
+    std::unique_ptr<smtliblogic::SMTLogicBlock> const smtLibLogic = std::make_unique<smtliblogic::SMTLogicBlock>(false, ss);

```

Comment:
```suggestion
    auto const smtLibLogic = std::make_unique<smtliblogic::SMTLogicBlock>(false, ss);
```
makes indentation down below so much nicer. Also consider that change for the other tests.